src/pages/About.scss

```diff
@@ -94,3 +94,10 @@
   }
 
 }
+
+#date-input-popover {
+  --offset-y: -var(--ion-safe-area-bottom);
+
+  --max-width: 90%;
```

Comment:
What does the max width do here? Removing it made no difference on my end

Reply:
It's a failsafe for old phones with a width < 336px, so the popover always fits within the screen.